src/formatDate.js

```diff
@@ -50,7 +53,66 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const YEAR_FORMAT_YY = 'YY';
+  const YEAR_FORMAT_YYYY = 'YYYY';
+  const DAY_FORMAT = 'DD';
+  const MONTH_FORMAT = 'MM';
+  const getPrefixYear = (yearBefore, newFormat) => {
+    const yearUpdateFormat
+      = newFormat.find(partDate => partDate.includes(YEAR_FORMAT_YY));
+
```

Comment:
Don't create functions inside other functions.
You can move it outside, also, constants too.